### ISSUE TITLE
add maps support?

### ISSUE THREAD

**Issue by benoitc** [cauterize-tools/caut-erl-ref#4]
Is thos possible to integrate maps support or is the order required ?

**Comment by Vagabond** [comment]
Yes, we could probably switch unions/records to use maps instead of proplists.
